packages/server/tests/acceptance/rpc.spec.ts

```diff
@@ -316,6 +323,11 @@ describe('RPC Server Acceptance Tests', function() {
             expect(res).to.be.equal('0x0');
         });
 
+        it('should execute "eth_getTransactionCount" for account with non-zero nonce with alais', async function () {
```

Comment:
We do not actually need this test. In `should execute "eth_getTransactionCount" for account with non-zero nonce` we are querying the `transactionCount` for `account.address` which is a alias created account.